frontend/src/components/HostMetricsDetail/HostMetricsLogs/HostMetricLogs.styles.scss

```diff
@@ -0,0 +1,46 @@
+.host-metrics-logs-container {
+    margin-top: 1rem;
+
+    .filter-section {
+       width: 35.7rem;
+    }
+
+    .host-metrics-logs-header {
+        display: flex;
+        justify-content: space-between;
+    }
+
+    .host-metrics-logs {
+        margin-top: 2rem;
+
+    .virtuoso-list {
+        overflow-y: hidden !important;
+
+        &::-webkit-scrollbar {
+            width: 0.3rem;
+            height: 0.3rem;
+        }
+
+        &::-webkit-scrollbar-track {
+            background: transparent;
+        }
+
+        &::-webkit-scrollbar-thumb {
+            background: var(--bg-slate-300);
+        }
+
+        &::-webkit-scrollbar-thumb:hover {
+            background: var(--bg-slate-200);
+        }
+
+        .ant-row {
+            width: fit-content;
+        }
+        }
+
+        .skeleton-container {
+            height: 100%;
+            padding: 16px;
+          }
+    }
+}
```

Comment:
lightMode missing

Reply:
@YounixM Added the light mode styles that were missing.